Makefile.am

```diff
@@ -234,12 +238,15 @@ libcupsfilters_la_CFLAGS = \
 	$(LCMS_CFLAGS) \
 	$(LIBQPDF_CFLAGS) \
 	$(LIBJPEG_CFLAGS) \
+	$(LIBJXL_CFLAGS) \
 	$(EXIF_CFLAGS) \
 	$(LIBPNG_CFLAGS) \
+	$(shell pkg-config --cflags libjxl) \
```

Comment:
Please remove this line. Once you have already the `$(LIBJXL_CFLAGS)` 3 lines above this one, and second, do not run `pkg-config` in `Makefile.am`, as you already have run it and defined variables with the results in `configure.ac`.